src/wireviz/Harness.py

```diff
@@ -646,6 +729,12 @@ def typecheck(name: str, value: Any, expect: type) -> None:
     # cache for the GraphViz Graph object
     # do not access directly, use self.graph instead
     _graph = None
+            
+    # This renders the graph with gvpr and neato, this is needed to be able to draw the stright lines for the jumpers
+    def graphRender(self, type, filename, graph):
+        graph.save(filename=f"{filename}_tmp.gv")
+        os.system(f"dot {filename}_tmp.gv | gvpr -q -cf pin2pin.gvpr | neato -n2 -T{type} -o {filename}.{type}")
```

Comment:
No check if the commands are available and that they are executed correctly.

Reply:
File actions and OS calls are unreliable, so you need to add some exceptions for these Ines.

Reply:
Added check with find_executable
https://stackoverflow.com/questions/11210104/check-if-a-program-exists-from-a-python-script
https://docs.python.org/3.9/distutils/apiref.html#module-distutils.spawn